src/comments.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
+
+const utils = require("prettier").util;
+
+const addDanglingComment = utils.addDanglingComment;
+
+/*
+Comment functions are meant to inspect various edge cases using given comment nodes,
+with information about where those comment nodes exist in the tree (ie enclosingNode,
+previousNode, followingNode), and then either call the built in functions to handle
+certain cases (ie addLeadingComment, addTrailingComment, addDanglingComment), or just
+let prettier core handle them. To signal that the plugin is taking over, the comment
+handler function should return true, otherwise returning false signals that prettier
+core should handle the comment
+
+args:
+  comment
+  text
+  options
+  ast
+  isLastComment
+*/
+
+const handleOwnLineComment = function(comment) {
+  if (
+    handleClass(comment) ||
+    handleFunction(comment) ||
+    handleForLoop(comment) ||
+    handleTryCatch(comment)
+  ) {
+    return true;
+  }
+  return false;
```

Comment:
Why not just `return handleClass(comment)  || ...` (no extra `if`)?

Reply:
good point 😄 i'll update now

Reply:
all set 👍